### ISSUE TITLE
Checkout error

### ISSUE THREAD

**Issue by mariusbolik** [woocommerce/woocommerce-gutenberg-products-block#7481]
## Describe the bug

I don't know why, but my checkout page stopped working. I'm using the Gutenberg Checkout block. I already disabled WooCommerce Payments and PayPal Payments. Still no progress. Also in WP Debug Mode I can't see any promising error message. I downgraded from 7.0.0 to 6.9.4. Still no success.

## To reproduce

1. Go to https://mrweedo.de
2. Add an item to the card
3. Click on 'Kasse' to see the checkout page

## Expected behavior

Checkout block should be displayed.

## Screenshots

<img width="1750" alt="Bildschirmfoto 2022-10-25 um 19 56 24" src="https://user-images.githubusercontent.com/39862171/197957788-37760157-9330-4fd8-8452-f3f3fdeca2b3.png">

<img width="1750" alt="Bildschirmfoto 2022-10-26 um 09 02 24" src="https://user-images.githubusercontent.com/39862171/197957939-8285af77-a8ea-4b0c-b9ee-1ddf8a3d0a37.png">

### WordPress (please complete the following information):

-   WordPress version: 6.0.3
-   WooCommerce version: 7.0.0
-   WooCommerce Blocks version: Using the integrated block of WC 7
-   Site language: German
-   Any other plugins installed: WooCommerce Payments, PayPal Payments

### Desktop (please complete the following information):

-   OS: macOS
-   Browser Opera
-   Version 91

**Comment by nielslange** [comment]
Hello @mariusbolik 👋

Thanks for reaching out. I could reproduce the problem while looking at your store. Via the translation, I traced back the component that is using this string and found the following section: https://github.com/woocommerce/woocommerce-blocks/blob/c7e41d60226d271fe9308f169678a74a0a0e3f6a/assets/js/blocks/checkout/checkout-order-error/index.js#L33-L69

The header of this component says: _"When an order was not created for the checkout, for example, when an item was out of stock, this component will be shown instead of the checkout form."_

This indicates that there might be a stock-related issue. Can you double-check the stock management of your products?

**Comment by mariusbolik** [comment]
Hi @nielslange, thank you for the fast response. I checked it. Sadly, no success. But it noticed that if I deactivate Elementor, the error is gone. I also disabled some Elementor settings, but the error only disappears, if the plugin is fully deactivated. I really need Elementor for my site. Is there anything else I can do?

**Comment by nielslange** [comment]
>  I checked it. Sadly, no success. But it noticed that if I deactivate Elementor, the error is gone. I also disabled some Elementor settings, but the error only disappears, if the plugin is fully deactivated. I really need Elementor for my site. Is there anything else I can do?

Thanks for running additional tests. I just went ahead and created the following test site: https://scrawny-penguin.jurassic.ninja/

These are the specification of this site:

- WordPress 6.0.3 (core)
- WooCommerce 7.0.0 (plugin)
- WooCommerce Blocks 8.8.0 (plugin)
- Elementor 3.7.8 (plugin)
- Hello Elementor 2.6.1 (theme)

The test site only contains the following pages:

- Shop: Created using the WordPress Editor showing the All Products block
- Cart: Created using the WordPress Editor showing the Cart block
- Checkout: Created using the WordPress Editor showing the Checkout block 

With this setup, I was able to both access the Checkout page and to place an order. If you're using the free Elementor plugin, I suggest trying to set up this scenario, e.g. using https://tastewp.com/, which allows the creation of disposable WordPress sites. In case you are using Elementor Pro, I suggest reaching out to the Elementor support via https://elementor.com/support/ to report this case.
